server/wallet/claim-rewards.go

```diff
@@ -85,22 +85,41 @@ func (c *walletClaimRewardsContext) PrepareData(data *swapi.WalletClaimRewardsDa
 	data.TokenName = token.Name()
 	data.TokenSymbol = token.Symbol()
 
+	// Get the native token address
+	err = qMgr.Query(func(mc *batch.MultiCaller) error {
+		splitWarehouseContract.NativeToken(mc, &data.NativeToken)
+		return nil
+	}, nil)
+	if err != nil {
+		return types.ResponseStatus_Error, fmt.Errorf("error querying claimable rewards: %w", err)
+	}
+
 	// Get the claimable rewards
 	err = qMgr.Query(func(mc *batch.MultiCaller) error {
-		splitMainContract.GetErc20Balance(mc, &data.WithdrawableToken, nodeAddress, *res.Vault)
-		splitMainContract.GetEthBalance(mc, &data.WithdrawableEth, nodeAddress)
+		splitWarehouseContract.BalanceOf(mc, &data.WithdrawableToken, nodeAddress, *res.Vault)
+		splitWarehouseContract.BalanceOf(mc, &data.WithdrawableEth, nodeAddress, data.NativeToken)
```

Comment:
We're probably going to have to rename this to like `WithdrawableNativeToken` instead of `WithdrawableEth` if they're generalizing it to work on other networks, just in case we ever move this over to L2s. The CLI will have to handle changing their name based on whatever the native token's name ends up being.

Reply:
Fixed